frontend/app/(footer)/fulfilment/[fulfilmentSessionId]/[fulfilmentEntityId]/page.tsx

```
@@ -0,0 +1,135 @@
+      const handleNext = async () => {
+        try {
+          const nextUrl = await getNextFulfilmentEntityUrl(params.fulfilmentSessionId, params.fulfilmentEntityId);
+          if (nextUrl) {
+            router.push(nextUrl);
+          } else {
+            fulfilmentSessionEntityPageLogger.info("No next fulfilment entity available");
+          }
+        } catch (error) {
+          fulfilmentSessionEntityPageLogger.error(`Error navigating to next entity: ${error}`);
+        }
+      };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)